index.test.js

```diff
@@ -1073,6 +1079,170 @@ describe('Deploy to ECS', () => {
         expect(mockEcsUpdateService).toHaveBeenCalledTimes(0);
     });
 
+    test('deployment is sucessful with rolloutState in COMPLETED', async () => {
+        core.getInput = jest
+            .fn()
+            .mockReturnValueOnce('task-definition.json')  // task-definition
+            .mockReturnValueOnce('service-456')          // service
+            .mockReturnValueOnce('cluster-789')          // cluster
+            .mockReturnValueOnce('false')                // wait-for-service-stability
+            .mockReturnValueOnce('')                     // wait-for-minutes
+            .mockReturnValueOnce('false')                 // force-new-deployment
+            .mockReturnValueOnce('true')                 // show-service-events
+            .mockReturnValueOnce(1);                     // show-service-events-frequency
+
+        mockEcsDescribeServices.mockImplementation(() => {
+            return {
+                promise() {
```

Comment:
The mock needs to be updated as the action is now using JS SDK v3

Refer: https://github.com/aws-actions/amazon-ecs-deploy-task-definition/pull/529

Reply:
@trivikr I could resolve the conflicts but the tests I implemented are failing, can I have some guidance on how should I correct them? I am more a cloud engineer than a javascript developer, so I am noob where talking about testing code